### PR TITLE
Add Django Guestbook template

### PR DESCRIPTION
**Pull request by wangxf123456**
This one is similar with the Hello World template, but a Postgres database is included.

### CODE REVIEW

**Comment by wangxf123456** [comment]
@simonz130 @brianquinlan @enriquejosepadilla

**Comment by wangxf123456** [comment]
@simonz130 Friendly ping.

File: python/django/python-guestbook/kubernetes-manifests/guestbook-app.deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        app: python-guestbook
+        tier: app
+    spec:
+      initContainers:
+      # Wait for the database to get ready before starting the server, otherwise
+      # the server will crash
+      - name: check-db-ready
+        image: postgres
+        command: ['sh', '-c',
+          'until pg_isready -h python-guestbook-db.default.svc.cluster.local -p 5432;
```
Comment (simonz130) [inline python/django/python-guestbook/kubernetes-manifests/guestbook-app.deployment.yaml]: can you replace the port with environment variable holding the port number? I see you have the port in DATABASE_PORT

File: python/django/python-guestbook/kubernetes-manifests/guestbook-app.migration.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  template:
+    metadata:
+      name: python-guestbook-migration
+    spec:
+      initContainers:
+      # Wait for the database to get ready before doing migration
+      - name: check-db-ready
+        image: postgres
+        command: ['sh', '-c',
+          'until pg_isready -h python-guestbook-db.default.svc.cluster.local -p 5432;
```
Comment (simonz130) [inline python/django/python-guestbook/kubernetes-manifests/guestbook-app.migration.yaml]: ditto

**Comment by simonz130** [comment]
> @simonz130 Friendly ping.

Sorry for the delay, I was out for 2 weeks and now back.
LGTM from me but I also asked Charlie, our Python guru to take a second look.

File: python/django/python-guestbook/kubernetes-manifests/guestbook-app.deployment.yaml (hunk 1 above)
Comment (wangxf123456) [reply python/django/python-guestbook/kubernetes-manifests/guestbook-app.deployment.yaml]: Thanks very much for your feedback. I tried to replace "5432" with "$DATABASE_PORT", but looks like "containers" and "initContainers" are in different environments. Those environment variables are only available in "containers" but not in "initContainers". I tried to find whether there are any ways to define environment variables for both "containers" and "initContainers" but was not able to find any.

Comment (simonz130) [reply python/django/python-guestbook/kubernetes-manifests/guestbook-app.deployment.yaml]: OK,  let's leave it as 5432 then. Thanks for checking the options.

**Pull request merged**